.github/workflows/prbot.yml

```diff
@@ -1,18 +1,155 @@
 name: PR Robot
-on:
-- pull_request
-
+on: [pull_request]
 jobs:
-  pr-robot:
+  build:
+    name: "Build"
     runs-on: ubuntu-latest
     env:
       GITHUB_CREDENTIALS: ${{ secrets.GITHUB_TOKEN }}
     steps:
-    - uses: actions/checkout@v2
-      with:
+      - uses: actions/checkout@v2
+        with:
           fetch-depth: 0
-    - uses: actions/setup-node@v1
-      with:
-          node-version: '14'
-    - run: npm i
-    - run: npm run pr:report
+      - uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+      - uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+      - run: |
+          npm ci
+          npx lerna bootstrap
```

Comment:
Would it be faster to run `npx lerna bootstrap --ci` ([doc](https://github.com/lerna/lerna/blob/main/commands/bootstrap/README.md#--no-ci)) instead of `npm ci` followed by bootstrapping? We might need to install `lerna` though

Reply:
woops, I meant to put `npx lerna bootstrap --ci`, but accidentally removed the `--ci` parameter at some point.

As for skipping `npm ci`, we do need to install lerna, but I guess we could install it globally to shorten the build a little.